prac_07/language_file_reader.py

```diff
@@ -26,9 +26,10 @@ def main():
         # print(parts)  # debugging
         # Reflection is stored as a string (Yes/No) and we want a Boolean
         reflection = parts[2] == "Yes"
+        pointer_arithmetic = parts[3] == "Yes"
         # Construct a ProgrammingLanguage object using the elements
         # year should be an int
-        language = ProgrammingLanguage(parts[0], parts[1], reflection, int(parts[3]))
+        language = ProgrammingLanguage(parts[0], parts[1], reflection, pointer_arithmetic, int(parts[4]))
```

Comment:
Avoid Hardcoding Indexes in CSV Parsing:

Using hardcoded indexes (like parts[3] and parts[4]) can lead to errors if the file structure changes. Consider using named indices or converting the CSV data into a namedtuple for improved readability.